CHANGELOG.md

```diff
@@ -7,11 +7,34 @@ This projects adheres to [Keep a CHANGELOG](http://keepachangelog.com/).
 Up to version 8.0.0, the `major.minor` version numbers were based on the PHP version for which compatibility check support was added, with `patch` version numbers being specific to this library.
 From version 8.0.0 onwards, [Semantic Versioning](http://semver.org/) is used.
 
+**IMPORTANT**: The 8.0.0 release contains a **breaking change**. Please read the below information carefully before upgrading!
+
 
 ## [Unreleased]
 
 _Nothing yet._
 
+## [8.0.1] - 2017-08-07
+
+See all related issues and PRs in the [8.0.1 milestone].
+
```

Comment:
I've tried to be consistent with the emojis used for entries up to now so let me document the ones I've used and for what. Generally, they have also been used in this same order and for a group of the same "kind" of changes, the order has been based on the sniff name:

#### In Added:
:zap: `:zap:` - new feature for the library, like adding the ability to specify `testVersion`
:two_hearts: `:two_hearts:` - support for a higher PHPCS version which contains breaking changes ;-)
:star2: `:star2:` - a new sniff has been added
:star:  `:star:` - something new has been added to an existing sniff, like, for example, new functions to be checked for in the `DeprecatedFunctions` sniff or a new utility method has been added

#### In Changed:
:pushpin: `:pushpin:` - structural improvement to an existing sniff, like changing something from warning to error (up the error level) or fixing a check to function independently of code style, or adding additional detection capabilities for the same to a sniff.
:twisted_rightwards_arrows: `:twisted_rightwards_arrows:` - moving a check from one sniff to another or renaming of a sniff
:rewind: `:rewind:` - fixes for backward compatibility with older PHPCS versions and older PHP versions
:pencil2: `:pencil2:` - minor change, like fixing the grammar of an error message or changing something from error to warning (downgrade)

#### In Removed:
:no_entry_sign: `:no_entry_sign:` - removal of something, a sniff or otherwise

#### In Fixed:
:bug: `:bug:` - Bug fixes ;-)
:white_check_mark: `:white_check_mark:` - if a PHPCS compatibility fix for new PHPCS versions was made

#### Usage across sections:
:fire: `:fire:` - Fixing a serious bug, i.e. a bug which was causing incorrect results across the board and/or for breaking changes introduced by PHPCompatibility
:recycle: `:recycle:` - code refactoring, including new utility methods being added to replace duplicate code elsewhere
:books: `:books:` - anything to do with the user facing documentation
:umbrella: `:umbrella:` - adding/fixing unit tests
:wrench: `:wrench:` - anything to do with the tooling around the standard, Composer, PHPCS, PHPUnit, Coveralls, Scrutinizer, Travis etc
:green_heart: `:green_heart:` - improvements to the build verification process like testing against a new PHP version or fixing a failing build due to changes in newer PHPCS versions